backend/cmd/server.go

```diff
@@ -14,7 +44,7 @@ func main() {
 	insecure := flag.Bool("insecure-ssl", false, "Accept/Ignore all server SSL certificates")
 	// @todo: Make this a uint and validate the values
 	port := flag.String("port", "4466", "Port to listen from")
-	pluginDir := flag.String("plugins-dir", "./plugins", "Specify the plugins directory to build the backend with")
+	pluginDir := flag.String("plugins-dir", "", "Specify the plugins directory to build the backend with")
```

Comment:
Nitpick: In order for it to inform users about what the default dir is (without having to check the docs), we could get the default dir and use it here as the default.

Reply:
Good idea. This is done.